app/feature/feature-insurance-certificate/src/main/kotlin/com/hedvig/android/feature/insurance/certificate/data/GetInsuranceEvidenceInitialDataUseCase.kt

```diff
@@ -0,0 +1,24 @@
+package com.hedvig.android.feature.insurance.certificate.data
+
+import arrow.core.Either
+import arrow.core.raise.either
+import com.apollographql.apollo.ApolloClient
+import com.hedvig.android.apollo.ErrorMessage
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.core.common.ErrorMessage
+import octopus.InsuranceEvidenceInitialDataQuery
+
+internal interface GetInsuranceEvidenceInitialDataUseCase {
```

Comment:
Hmm, do you think we can mention the fact that the data is `email` specifically, and not just "data"?